app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/Account.kt

```diff
@@ -161,7 +161,7 @@ class Account : MifosBaseFragment() , AccountsContract.View {
     }
 
 
-    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
```

Comment:
You seemed to have removed the nullability property for many function parameters. Are you sure that there will not be any side effects (like NPEs) because of this?

Reply:
Hover SDK was raising errors for them. So, I have to change them.